liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/BasicSupport.java

```diff
@@ -163,6 +163,12 @@
     @Parameter(alias="serverEnv", property = "serverEnvFile")
     protected File serverEnvFile;
 
+    /**
+     * Version to override any specified in the assemblyArtifact
+     */
+    @Parameter(alias = "libertyRuntimeVersion", property = "liberty.runtime.version")
+    protected String libertyRuntimeVersion = null;
```

Comment:
Have you tested that the property is recognizable as a Maven property? The invoker test uses `-D` which is a for a system property. I'm not sure if it sets it as a Maven property as well.

Reply:
I would also rename this parameter to be `runtimeVersion`. The user already knows that it is a Liberty runtime. I would suggest this for the property as well unless Alasdair specifically wants it to be called `liberty.runtime.version`.

Reply:
I can add another test to make sure the Maven parameter works in the pom.xml also. As for the name, they explicitly asked for a property called liberty.runtime.version in the gitter channel (Emily, Sebastian, Alasdair). So should I keep the Maven parameter as libertyRuntimeVersion for consistency with the property name? Or does it matter?

Reply:
We can keep it as is.